.github/ISSUE_TEMPLATE/alm-accelerator-for-advanced-makers-bug-report.md

```diff
@@ -1,17 +1,23 @@
 ---
-name: Bug report
-about: Create a report to help us improve the CoE Starter Kit
-title: ''
-labels: ''
-assignees: ''
+name: "[ALM Accelerator] Bug report"
+about: Create a report to help us improve the ALM Accelerator for Advanced Makers
+title: "[ALM Accelerator - BUG]: issue title"
+labels: alm-accelerator
+assignees: 
 
 ---
 
 **Describe the bug**
 A clear and concise description of what the bug is.
 
+**Component (please tell us which part you are experiencing issues with):**
+
+- Version [e.g. 1.29]
+- Component [e.g. ALM Accelerator Canvas App]
+
```

Comment:
Add ALM Accelerator Pipelines/Workflows to the e.g.

Reply:
added this in latest commit